pyomo/contrib/doe/doe.py

```diff
@@ -180,15 +169,12 @@ def stochastic_program(
 
         Parameters
         -----------
-        design_values:
-            a ``dict`` where keys are design variable names, values are a dict whose keys are time point
-            and values are the design variable value at that time point
         if_optimize:
             if true, continue to do optimization. else, just run square problem with given design variable values
         objective_option:
-            supporting maximizing the 'det' determinant or the 'trace' trace of the FIM
-        jac_involved_measurement:
-            the measurement class involved in calculation. If None, take the overall measurement class
+            choose from the ObjectiveLib enum,
+            "det": maximizing the determinant with ObjectiveLib.det,
+            "trace": or the trace of the FIM with ObjectiveLib.trace
         scale_nominal_param_value:
             if True, the parameters are scaled by its own nominal value in param_init
         scale_constant_value:
```

Comment:
This is because for `scale_nominal_param_value`, the user only need to choose if they want to time the sensitivity by the parameter value; for `scale_constant_value`, she needs to choose to time it with a scalar number like 100, or 0.001. So if she decides not use `scale_constant_value`, she can just use the default option 1